### ISSUE TITLE
Installation instructions are confusing.

### ISSUE THREAD

**Issue by JHamCell** [openipcamera/openipc-firmware#31]
"With button still depressed, plug in power to the camera while continuing to hold for another 10 - 12 seconds."

Change to:

`With button still pressed, plug the power cable into the camera.`
`Continue to press the button for another 10 - 12 seconds until the connection light blinks back and forth between orange and blue.`
`Release the button.`

**If the light doesn't blink back and forth between orange and blue, what is the correct course of action?**

"Press the button again and you should hear a Chinese voice prompt if successful."

**What needs to be done if the chinese voice is not heard?**

This line is very confusing:
Remove SDcard and reboot camera and reinsert after boot until you hear "click click".

Remove SDcard
Reboot camera
Reinsert (SDcard?) after boot
Until you hear "click click" - **Do what until you hear "click click"?**
**What do I do when I hear click click?**

Repeat Step 5 until you hear "click click"
**Do I just keep keep reseting the camera and insert & ejecting the SDcard?**
When do I get to use the camera?

"Run openipscan.sh or openipscanwin.sh if you're on windows/mac and find the IP address of your new OpenIP Camera. Note: MAC address will be changed to c0:6d:1a*"
I tried running this in MobaXterm shell on Windows and the script needs nmap.
I loaded nmap on my linux sandbox VM and the script ran, but did not find my camera - which is still blinking orange & blue.

**Comment by skorokithakis** [comment]
I agree, I just got the camera and couldn't figure out how to make everything work. I am having exactly the same problems as above.

**Comment by openipcamera** [comment]
V2 is not supported on v.0.2.4

**Comment by skorokithakis** [comment]
:(

What would need to be done for this to be ported? I can help/test.

**Comment by openipcamera** [comment]
You need to get your camera to load the wifi script on first boot (the first click click) then reboot camera with sdcard inserted.

V1 will no longer be supported now that v2 is out and v1 is impossible to get more of.

**Comment by openipcamera** [comment]
There is a v2 alpha

**Comment by skorokithakis** [comment]
How can I try the v2 alpha? Do I need the `demo.bin` to get the camera to boot the wifi script?

Also, what is the initial file script filename that gets loaded? I'd be happy to help test the v2 alpha if you can give me some more details on how to load it.

**Comment by phantomdarkness** [comment]
Are the instructions for v2 identical to those for v1?  I tried installing the alpha last night and couldn't get it to work.  I held the button for 10-12 seconds, plugged in, kept holding for 10-12 seconds and couldn't get the light to blink.

**Comment by skorokithakis** [comment]
No, they're different, you can see the instructions for v2 on the Dafang hacks repo. I tried it and it worked perfectly.

**Comment by phantomdarkness** [comment]
Ok, tried that, but I'm confused how to follow the instructions [here ](https://github.com/EliasKotlyar/Xiaomi-Dafang-Hacks/blob/master/hacks/install_cfw.md) because there's no motor on the base so idk when step 7 is done or how to confirm like step 9 indicates.

**Comment by nniillaa** [comment]
@phantomdarkness , I'm having the same issue and cant seem to get this done. 
Could you please tell me what you did to get this all up and running?

**Comment by phantomdarkness** [comment]
@nniillaa try here: https://github.com/openipcamera/openipc-firmware#installation-instructions-wyze-v2

**Comment by claell** [comment]
@JHamCell This is still open, so are the problems still there? You can try and fix some of them yourself by creating a PR for the Readme here on GitHub including the changes.

**Comment by slate0** [comment]
I would appreciate it if the instructions noted specifically that they do not work with the WYZE V2.

**Comment by claell** [comment]
@slate0 You could create a PR including the note for that.
